tests/integration/tier0/custom-kernel/test_cutsom_kernel.py

```diff
@@ -0,0 +1,109 @@
+import os
+
+import pytest
+
+
+SYSTEM_RELEASE = os.environ["SYSTEM_RELEASE"]
```

Comment:
I believe here it should be:

```suggestion
SYSTEM_RELEASE = os.environ.get("SYSTEM_RELEASE")
```

Reply:
applied.

Reply:
Either or works fine, we use normal array access in most cases